books/serializers.py

```diff
@@ -0,0 +1,59 @@
+from rest_framework import serializers
+
+from books.models.books import Book, Author, Publisher
+from books.models.country import Country
+
+
+class BookSerializer(serializers.ModelSerializer):
+    id = serializers.IntegerField(read_only=True)
+    publisher = serializers.SlugRelatedField(slug_field='name', read_only=True)
+    country = serializers.SlugRelatedField(slug_field='name', read_only=True)
+    authors = serializers.SlugRelatedField(slug_field='name', many=True, read_only=True)
+
+    class Meta:
+        model = Book
+        fields = ("id", "name", "isbn", "authors", "number_of_pages", "publisher", "country", "release_date")
+
+    def to_representation(self, instance):
+        data = super(BookSerializer, self).to_representation(instance)
```

Comment:
@xhahid43eb Why you need to override this method if there isn't anything other than calling super?

Reply:
My bad, I forget to remove this function